doc/releases/changelog-dev.md

```diff
@@ -86,37 +111,43 @@
 
   ```pycon
   >>> bits, recipes = circuit()
+  >>> bits
   tensor([[0, 0],
           [1, 0],
-          [1, 0],
-          [0, 0],
           [0, 1]], dtype=uint8, requires_grad=True)
   >>> recipes
   tensor([[2, 2],
-          [0, 2],
-          [1, 0],
           [0, 2],
           [0, 2]], dtype=uint8, requires_grad=True)
   ```
 
-* Added the ``shadow_expval`` measurement for differentiable expectation value estimation using classical shadows.
+  - QNodes returning `qml.shadow_expval` yield the expectation value estimation using classical shadows:
 
   ```python
-  H = qml.Hamiltonian([1., 1.], [qml.PauliZ(0) @ qml.PauliZ(1), qml.PauliX(0) @ qml.PauliX(1)])
-
   dev = qml.device("default.qubit", wires=range(2), shots=10000)
+
   @qml.qnode(dev)
-  def qnode(x, H):
+  def circuit(x, H):
       qml.Hadamard(0)
       qml.CNOT((0,1))
       qml.RX(x, wires=0)
       return qml.shadow_expval(H)
+  ```
 
-  x = np.array(0.5, requires_grad=True)
-
-  print(qnode(x, H), qml.grad(qnode)(x, H))
+  ```pycon
+  >>> x = np.array(0.5, requires_grad=True)
+  >>> H = qml.Hamiltonian(
+      [1., 1.], 
+      [qml.PauliZ(0) @ qml.PauliZ(1), qml.PauliX(0) @ qml.PauliX(1)]
+  ) 
+  >>> circuit(x, H), 
+  tensor(1.8486, requires_grad=True) 
+  >>> qml.grad(circuit)(x, H))
+  -0.4797000000000001
   ```
 
+<h4>Operator stuff (TODO: IMPROVE TITLE)</h4>
```

Comment:
I put some things in this subsection, but it feels a little all over the place.

Reply:
Can't comment on it 😢 

line 165 (`qml.exp`):

Was this not a part of v0.25?